operator/mysqlcontroller/pipeline.go

```diff
@@ -0,0 +1,23 @@
+package mysqlcontroller
+
+import (
+	"github.com/crossplane/crossplane-runtime/pkg/event"
+	exoscalesdk "github.com/exoscale/egoscale/v2"
+	"sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+// Pipeline provisions IAMKeys on exoscale.com
+type Pipeline struct {
```

Comment:
```suggestion
type pipeline struct {
```

same nit here about the visibility and same caveat :smile: